src/smarthr-ui-preset.ts

```diff
@@ -26,6 +26,7 @@ export default {
       link: defaultColor.TEXT_LINK,
       background: defaultColor.BACKGROUND,
       border: theme('colors.grey.20'),
+      brand: defaultColor.BRAND,
```

Comment:
記述が揃ってなくて申し訳ないけど、colors に定義してあるので参照するとよさそう。

```suggestion
      brand: theme('colors.brand'),
```